src/util/safelywritablefile.cpp

```diff
@@ -263,6 +271,52 @@ bool SafelyWritableFile::commit() {
     return true;
 }
 
+bool SafelyWritableFile::useTempFileWithPrefix(const QString& coverArtSuffix) {
+    DEBUG_ASSERT(m_tempFileName.isNull());
+    if (!QFileInfo(m_origFileName).isWritable()) {
+        kLogger.warning()
+                << "Failed to prepare file for writing:"
+                << m_origFileName
+                << "is not writable.";
+        return false;
+    }
+
+    QString origFilePath = QFileInfo(m_origFileName).canonicalPath();
+    QString origFileCompleteBasename = QFileInfo(m_origFileName).completeBaseName();
+    QString tempFileCompleteBasename = kSafelyWritableTempFilePrefix +
+            origFileCompleteBasename + '.' + coverArtSuffix;
+    QString tempFileName = origFilePath + '/' + tempFileCompleteBasename;
+
+    QFile origFile(m_origFileName);
+    if (!origFile.copy(tempFileName)) {
```

Comment:
This function must not copy. It should calculate the temp file name only and store it as m_tempFileName.

Reply:
That comment was for `useTempFileWithPrefix()`. This function is now not used in `useTempFileWithPrefix()`. I will mark this as resolved if that's okay.